### DISCUSSION TITLE
Interest for a technical workshop for developers in London on Thursday November 9th

### DISCUSSION THREAD

**Discussion by bertfrees** [daisy/pipeline#721] [Meetings]
As already mentioned in https://github.com/daisy/pipeline/discussions/695, we're planning to organize a technical workshop for developers in London, on Thursday November 9th, the day before the [DAISY Technical Meeting](https://daisy.org/technical-meeting-2023/) and during the Accessible Publishing Conference. We would like to know who of you is interested in joining.

These are some suggested topics, but you are of course completely free to determine the agenda yourself:

- an explanation on how to create a module + migrate a Pipeline 1 script to Pipeline 2 as exercise
- an explanation on how to use the web API + write a client (library) as exercise
- an explanation on how to use the Java API
- how to read stack traces in the detailed log and how to locate functions in the source code
- explore the collection of braille style sheets (one for each language/region) within the Pipeline 2 core: how to organise it, how to maintain it

**Comment by bertfrees** [comment]
Update: we are proceeding with this one-day coding workshop. We have booked a room at the RNIB premises, on Thursday, November 9, from 9 AM to 4 PM. Please confirm if you are coming.

**Reply by egli** [reply]
I will be there

**Reply by josteinaj** [reply]
I will unfortunately not be there.
